liquid-fixpoint.cabal

```diff
@@ -144,6 +145,7 @@ library
                   , unordered-containers
                   , aeson
                   , rest-rewrite >= 0.1.1
+                  , time
```

Comment:
Probably we should be using the `clock` package instead of `time` since we don't care of time zones or days.